src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -198,8 +198,14 @@ def process_premiere_project(filepath, raven_client, vs_pathmap=None, db=None, c
     lg.debug("looking for referenced media....")
     for filepath in pp.getReferencedMedia():
         total_files += 1
-        lg.debug("Looking up {0}".format(filepath))
-        server_path = re.sub(u'^/Volumes', '/srv', filepath).encode('utf-8')
+        try:
+            lg.debug("Looking up {0}".format(filepath))
+        except UnicodeEncodeError:
+            lg.debug("Looking up {0}".format(filepath.encode('utf-8')))
+        try:
+            server_path = re.sub(u'^/Volumes', '/srv', filepath).encode('utf-8')
+        except UnicodeDecodeError:
```

Comment:
this is UnicodeDecodeError wherase all of the others are UnicodeEncodeError. Is that intentional?

Reply:
Yes.